lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -292,31 +383,93 @@ class MockTimers {
     }
   }
 
-  enable(timers = SUPPORTED_TIMERS) {
+  enable(...args) {
     if (this.#isEnabled) {
-      throw new ERR_INVALID_STATE(
-        'MockTimers is already enabled!',
-      );
+      throw new ERR_INVALID_STATE('MockTimers is already enabled!');
+    }
+
+    if (args.length <= 0) {
+      this.#timersInContext = SUPPORTED_TIMERS;
+      this.#now = 0;
+      this.#toggleEnableTimers(true);
+      return;
     }
 
-    validateArray(timers, 'timers');
+    let timersDefined = false;
+    // Checks if the first argument is the timer array or the initial time
+    if (ArrayIsArray(args[0])) {
+      validateArray(args[0], 'timers');
+      // Check that the timers passed are supported
+      ArrayPrototypeForEach(args[0], (timer) => {
+        if (!ArrayPrototypeIncludes(SUPPORTED_TIMERS, timer)) {
+          throw new ERR_INVALID_ARG_VALUE(
+            'timers',
+            timer,
+            `option ${timer} is not supported`,
+          );
+        }
+      });
+
+      this.#timersInContext = args[0];
+      timersDefined = true;
+      this.#now = 0;
+    } else if (this.#isValidDateWithGetTime(args[0])) {
+      // First argument is the initial time as Date
+      this.#now = args[0].getTime();
```

Comment:
do we mean to call `.getTime()` on the argument, or should this instead be:
```suggestion
      this.#now = DatePrototypeGetTime(args[0]);
```

Reply:
It's the intended one, if the first argument is an instance of `Date` then we will call the `getTime` method on it to get the timestamp

Reply:
right but if it's an actual Date object - which may not be `instanceof Date` - you don't want to rely on the presence of `Date.prototype.getTime` at runtime.